docs/default-fake-behavior.md

```diff
@@ -49,6 +49,18 @@ This behavior can be used to
 * supply values for the system under test to use (via the getter) or to
 * verify that the system under test performed the `set` action on the Fake
 
+### Object members
+
+Virtual methods inherited from `System.Object` are faked with a special default behavior:
+
+* `Equals` uses reference equality: it returns `true` if the argument is the fake
+  itself, `false` for any other argument.
+* `GetHashCode` returns the hashcode of its `FakeManager`.

```

Comment:
Technically true, but I wonder if it's more than we need to tell people. What if we want to change the implementation?
Perhaps it's just enough to indicate that it's consistent with our implementation of `Equals`? Who even knows what a `FakeManager is`!

Reply:
If you want to keep as is, I'm okay with it. If you want to change, feel free to amend rather than fixup.

Reply:
Yeah, I'm not too happy with this either...

> Perhaps it's just enough to indicate that it's consistent with our implementation of `Equals`?

I think I like it! I'll make the change

Reply:
Done